### DISCUSSION TITLE
셀카모드를 만드는데 파일 저장이 잘 안되어 질문드립니다.

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#367] [5. Q&A]
![셀카저장 2](https://user-images.githubusercontent.com/73985764/153553371-beadda46-7ef8-4664-8628-6d15143e632d.JPG)
위와 같은 오류가 생기면서 스크린샷을 따로 저장할수가 없네요... 
![셀카저장](https://user-images.githubusercontent.com/73985764/153553366-ec95d92c-d8a3-4952-a988-292b3117ef78.JPG)
코드는 다음과 같습니다. 저장장소는 딱히 구애받지 않으니 루트는 크게 상관없습니다. 어떻게 해결해야할까요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재 File IO관련 기능은 내부 정책상 막아 놓은 상태입니다.

스크린샷 저장 기능은 내부적으로 고려를 하고 있는 내용이라서 이후 업데이트가 된다면 사용 부탁드립니다.

**Reply by ted002** [reply]
아이고...알겠습니다!

**Comment by minjujuu** [comment]
https://studio.zepeto.me/kr/guides/image-save-share-upload
이번에 이미지 저장/공유/피드 올리기 기능이 추가되었다고 합니다!
